src/components/item/Item.tsx

```diff
@@ -0,0 +1,60 @@
+import styled from '@emotion/styled';
+
+interface ItemProps extends React.HTMLAttributes<HTMLInputElement> {
+  type?: 'radio' | 'checkbox';
+  name?: string;
+  label?: string;
+  emjCode?: string;
+  labelSize?: 'small' | 'large';
+}
+
+const Item = ({ type, label, emjCode, labelSize = 'small', ...rest }: ItemProps) => {
+  return (
+    <>
+      <ItemInput type={type} id={label} emjCode={emjCode} {...rest} data-testid="item-input" />
```

Comment:
제 생각으로는 ....

```suggestion
interface ItemProps extends React.HTMLAttributes<HTMLInputElement> {
  type: 'radio' | 'checkbox';
  name?: string;
  label?: string;
  emjCode?: string;
  labelSize?: 'small' | 'large';
}

const Item = ({ type, label, emjCode, labelSize = 'small', ...rest }: ItemProps) => {
  return (
    <>
      <ItemInput type={type} id={label} emjCode={emjCode} {...rest} data-testid="item-input" />
```
이렇게 required 처리하거나

```suggestion
interface ItemProps extends React.HTMLAttributes<HTMLInputElement> {
  type: 'radio' | 'checkbox';
  name?: string;
  label?: string;
  emjCode?: string;
  labelSize?: 'small' | 'large';
}

const Item = ({ type = 'radio', label, emjCode, labelSize = 'small', ...rest }: ItemProps) => {
  return (
    <>
      <ItemInput type={type} id={label} emjCode={emjCode} {...rest} data-testid="item-input" />
```
default 값을 넣어줘야할 거 같아요!

물론 `<Item />` 으로 사용할 때 예상하신 대로 동작한다면 상관없어요 !!

Reply:
오호 감사합니다! ! ! type을 required 하니까 ItemLabel에 type 없다고 에러내서 default값을 넣겠습니당 ㅋㅋㅋ